docs/supported-technologies.asciidoc

```diff
@@ -49,22 +49,32 @@ NOTE: The page load transaction is measured relative to the `fetchStart` timesta
+Focusses on the initial rendering and measures the time from when the page starts loading to when any part of the page's content is displayed on the screen. The agent uses the https://www.w3.org/TR/paint-timing/#first-contentful-paint[Paint timing API] available in the browser to capture the timing information. Captured as transaction mark for `page-load` transaction for all chromium-based browsers (Chrome >60, Edge >79, Opera >47, etc.).
 
 `Largest contentful paint (LCP)`::
-A new page-load metric that denotes the time from when the page starts loading to when the critical element (largest text or image element) is displayed on the screen. Captured as transaction mark for `page-load` transaction.
+A new page-load metric that denotes the time from when the page starts loading to when the critical element (largest text, image, video elements) is displayed on the screen. LCP is available in the browser through 
+https://wicg.github.io/largest-contentful-paint/[LargestContentfulPaint API] which relies on the draft https://wicg.github.io/element-timing/[Element Timing API]. LCP is one of the https://web.dev/vitals/[core web vitals] metric and
+available only in Chrome >77. Captured as transaction mark for `page-load` transaction, maintain LCP within the first *2.5 seconds* of page-load to provide a good user experience.
 
 `First input delay (FID)`::
-The time from when a user first interacts with your site (mouse clicks, taps, select dropdowns, etc.) to the time when the browser is able to respond to that interaction. Captured as `First Input Delay` span for `page-load` transaction.
+FID quantifies the experience the user feels when trying to interact with the page. It is measured as the time between when a user first interacts with your site (mouse clicks, taps, select dropdowns, etc.) to the time when the
+browser can respond to that interaction. FID is one of the https://web.dev/vitals/[core web vitals] metric and available only in Chrome >85 via https://wicg.github.io/event-timing/[Event Timing API]. Captured as `First Input Delay` span for `page-load` transaction, maintain FID of *less than 100 milliseconds* to provide a good user experience.
 
 `Total blocking time (TBT)`::
-The sum of the blocking time (duration above 50 ms) for each long task that occurs between the First contentful paint and the time when the transaction is ended. Captured as `Total Blocking Time` span for `page-load` transaction.
+The sum of the blocking time (duration above 50 ms) for each long task that occurs between the First contentful paint and the time when the transaction is completed. https://web.dev/tbt/[Total blocking time] is a
+great companion metric https://web.dev/tti/[Time to interactive] (TTI) which is lab metric and not available in the field through browser APIs. The agent captures TBT based on the number of long tasks that occurred during the page load lifecycle. Captured as `Total Blocking Time` span for `page-load` transaction.
```

Comment:
```suggestion
great companion metric for https://web.dev/tti/[Time to interactive] (TTI) which is lab metric and not available in the field through browser APIs. The agent captures TBT based on the number of long tasks that occurred during the page load lifecycle. Captured as `Total Blocking Time` span for `page-load` transaction.
```

I think we can in addition to/instead of TTI mention that TBT can be used in the lab as proxy for FID.

Reply:
I didn't add that part, as its there in the external doc and some companies still use TBT as a hint in the field as it can be used to find CPU hogs.